File: programme/inches-to-feet/inches-to-feet.js
// Write a program to add two numbers
// Execute: node inches-to-feet.js

"use strict";

const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

rl.question("Input  : ", (inches) => {
  const result = parseInt(inches) / 12;
  if (!isNaN(result)) {
    console.log(`Output : ${result}`);
  } else {
    console.log("An Error occured");
  }
  rl.close();
});

File: programme/inches-to-feet/InchesToFeet.java
// Write a programme to convert inches to feet
import java.util.Scanner;

class InchesToFeet {

  public static void main(String[] args) {
    try (Scanner sc = new Scanner(System.in)) {
      System.out.print("Input  : ");
      int inches = sc.nextInt();
      System.out.print("Output : ");
      System.out.println(inches / 12.0);
    } catch (Exception e) {
      System.err.println("An Error occured");
    }
  }
}
